examples/tutorials/07_vpp-connection/main.go

```diff
@@ -0,0 +1,158 @@
+//  Copyright (c) 2019 Cisco and/or its affiliates.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at:
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package main
+
+import (
+	"github.com/ligato/vpp-agent/plugins/vpp/binapi/interfaces"
+	"log"
+	"net"
+	"time"
+
+	"git.fd.io/govpp.git/api"
+	"github.com/ligato/vpp-agent/plugins/govppmux"
+
+	"github.com/ligato/cn-infra/agent"
+)
+
+func main() {
+	// Create an instance of our plugin.
+	p := new(HelloWorld)
+	p.GoVPPMux = &govppmux.DefaultPlugin
+
+	// Create new agent with our plugin instance.
+	a := agent.NewAgent(agent.AllPlugins(p))
+
+	// Run starts the agent with plugins, wait until shutdown
+	// and then stops the agent and its plugins.
+	if err := a.Run(); err != nil {
+		log.Fatalln(err)
+	}
+}
+
+// HelloWorld represents our plugin.
+type HelloWorld struct {
+	vppChan api.Channel
+
+	GoVPPMux govppmux.API
+}
+
+// String is used to identify the plugin by giving it name.
+func (p *HelloWorld) String() string {
+	return "HelloWorld"
+}
+
+// Init is executed on agent initialization.
+func (p *HelloWorld) Init() (err error) {
+	log.Println("Hello World!")
+
+	if p.vppChan, err = p.GoVPPMux.NewAPIChannel(); err != nil {
+		panic(err)
+	}
+
+	go func() {
+		p.syncVppCall()
+		p.asyncVppCall()
+		p.multiRequestCall()
+		return
```

Comment:
The `return` does not do anything. 

Also you are still doing it in Init phase.. it's not good practice to run anything before agent and all its plugins started. It might not matter in this case.. but this is a tutorial and it should teach proper practices.

The most correct way would be to run all of these after agent started.. similarly to what I mentioned before: https://github.com/ligato/vpp-agent/blob/2e73aad7e489c089b95e7bd3f874244962112a95/tests/perf/grpc-perf/main.go#L67-L76
This way the example also exits cleanly when done. Since there is no need for it to keep running in this case.